src/interaction/LennardJonesSoftcoreTI.hpp

```diff
@@ -0,0 +1,281 @@
+/*
+  Copyright (C) 2012,2013,2016
```

Comment:
Actually large portions of these files are based on the previously existing LennardJones and ReactionField. I'm not sure we should delete references to Frauenhofer.